src/MeshInertiaCalculator.cc

```diff
@@ -40,6 +40,7 @@ using namespace sim;
 //////////////////////////////////////////////////
 void MeshInertiaCalculator::GetMeshTriangles(
   std::vector<Triangle>& _triangles,
+  gz::math::Vector3d _meshScale,
```

Comment:
make it const ref? i.e. `const gz::math::Vector3d &_meshScale`

Reply:
Updated in [d892b53](https://github.com/gazebosim/gz-sim/pull/2171/commits/d892b53865a8975f41867da148c89287f9f094b2)